### PR TITLE
docs: postGIS - add GDAL raster driver information

### PR DESCRIPTION
**Pull request by mkiyama**
How to enable GDAL raster drivers for postGIS

HTML - on GPDB doc review site
http://docs-gpdb-review-staging.cfapps.io/530/ref_guide/extensions/postGIS.html#topic_ydr_q5l_ybb

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 1]
```diff
@@ -123,7 +123,37 @@
...
         <codeblock>export GDAL_DATA=$GPHOME/share/gdal
 export POSTGIS_ENABLE_OUTDB_RASTERS=0
-export POSTGIS_GDAL_ENABLED_DRIVERS=ENABLE_ALL</codeblock>
+export POSTGIS_GDAL_ENABLED_DRIVERS=DISABLE_ALL</codeblock>
+      </body>
+    </topic>
+    <topic id="topic_ydr_q5l_ybb">
+      <title>Enabling GDAL Raster Drivers</title>
+      <body>
+        <p>PostGIS uses GDAL raster drivers when processing raster data with commands such a
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: "such a" -> "such as"

File: gpdb-doc/dita/ref_guide/extensions/postGIS.xml [hunk 2]
```diff
@@ -123,7 +123,37 @@
...
+            <codeph>ST_AsJPEG()</codeph>. As the default, PostGIS disables all raster drivers. You
+          enable raster drivers by setting the value of
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/extensions/postGIS.xml]: "of" -> "of the"

**Comment by mkiyama** [comment]
@dyozie - These changes are only for 5.x, 6.
4.3.x is still using an earlier version of postGIS

**Comment by mkiyama** [comment]
@jingyimei  did you have a chance to review?

**Comment by mkiyama** [comment]
@jingyimei 
The link seems to be working on the GPDB doc review site.
Let me know if I am missing something.

thanks

HTML - on GPDB doc review site
http://docs-gpdb-review-staging.cfapps.io/530/ref_guide/extensions/postGIS.html#topic_ydr_q5l_ybb

**Comment by jingyimei** [comment]
@mkiyama I tried again, and it worked this time. Now all looks good to me.

**Pull request merged**